src/schematic.rs

```diff
@@ -186,11 +187,11 @@ impl Default for Schematic {
         #[cfg(target_os = "macos")]
         {
             // retrieve libngspice.dylib from the following possible directories
-            let ret = Cmd::new("find")
+            let ret = process::Command::new("find")
                 .args(&["/usr/lib", "/usr/local/lib"])
                 .arg("-name")
                 .arg("*libngspice.dylib")
-                .stdout(Stdio::piped())
+                .stdout(process::Stdio::piped())
```

Comment:
this is my bad - I didn't realize process::* it was being used in here - feel free to import it however you like